### PR TITLE
chore(*): unify config options with python

### LINKED ISSUE #207: Rename config options to align with Python agent
We're renaming our config options on both this agent and the Python agent on options that are the same across.

The following config options needs to be renamed in the Node.js agent:

- `sourceContextErrorAppFrames` -> `sourceLinesErrorAppFrames`
- `sourceContextSpanAppFrames` -> `sourceLinesSpanAppFrames`
- `sourceContextErrorLibraryFrames` -> `sourceLinesErrorLibraryFrames`
- `sourceContextSpanLibraryFrames` -> `sourceLinesSpanLibraryFrames`
- `validateServerCert` -> `verifyServerCert`

### PR DESCRIPTION
**Pull request by Qard**
Fixes #207

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ var ENV_TABLE = {
   serviceName: 'ELASTIC_APM_SERVICE_NAME',
   secretToken: 'ELASTIC_APM_SECRET_TOKEN',
   serverUrl: 'ELASTIC_APM_SERVER_URL',
-  validateServerCert: 'ELASTIC_APM_VALIDATE_SERVER_CERT',
+  verifyServerCert: 'ELASTIC_APM_VALIDATE_SERVER_CERT',
```
Comment (watson) [inline lib/config.js]: s/VALIDATE/VERIFY/

File: lib/config.js [hunk 2]
```diff
@@ -65,16 +65,16 @@ var ENV_TABLE = {
...
   asyncHooks: 'ELASTIC_APM_ASYNC_HOOKS',
   ff_captureFrame: 'ELASTIC_APM_FF_CAPTURE_FRAME',
-  sourceContextErrorAppFrames: 'ELASTIC_APM_SOURCE_CONTEXT_ERROR_APP_FRAMES',
-  sourceContextErrorLibraryFrames: 'ELASTIC_APM_SOURCE_CONTEXT_ERROR_LIBRARY_FRAMES',
-  sourceContextSpanAppFrames: 'ELASTIC_APM_SOURCE_CONTEXT_SPAN_APP_FRAMES',
-  sourceContextSpanLibraryFrames: 'ELASTIC_APM_SOURCE_CONTEXT_SPAN_LIBRARY_FRAMES',
+  sourceLinesErrorAppFrames: 'ELASTIC_APM_SOURCE_CONTEXT_ERROR_APP_FRAMES',
+  sourceLinesErrorLibraryFrames: 'ELASTIC_APM_SOURCE_CONTEXT_ERROR_LIBRARY_FRAMES',
+  sourceLinesSpanAppFrames: 'ELASTIC_APM_SOURCE_CONTEXT_SPAN_APP_FRAMES',
+  sourceLinesSpanLibraryFrames: 'ELASTIC_APM_SOURCE_CONTEXT_SPAN_LIBRARY_FRAMES',
```
Comment (watson) [inline lib/config.js]: s/CONTEXT/LINES/

(also on the 3 lines above)

File: test/config.js [hunk 3]
```diff
@@ -16,10 +16,10 @@ var optionFixtures = [
...
   ['instrument', 'INSTRUMENT', true],
   ['asyncHooks', 'ASYNC_HOOKS', true],
-  ['sourceContextErrorAppFrames', 'SOURCE_CONTEXT_ERROR_APP_FRAMES', 5],
-  ['sourceContextErrorLibraryFrames', 'SOURCE_CONTEXT_ERROR_LIBRARY_FRAMES', 5],
-  ['sourceContextSpanAppFrames', 'SOURCE_CONTEXT_SPAN_APP_FRAMES', 5],
-  ['sourceContextSpanLibraryFrames', 'SOURCE_CONTEXT_SPAN_LIBRARY_FRAMES', 0],
+  ['sourceLinesErrorAppFrames', 'SOURCE_CONTEXT_ERROR_APP_FRAMES', 5],
+  ['sourceLinesErrorLibraryFrames', 'SOURCE_CONTEXT_ERROR_LIBRARY_FRAMES', 5],
+  ['sourceLinesSpanAppFrames', 'SOURCE_CONTEXT_SPAN_APP_FRAMES', 5],
+  ['sourceLinesSpanLibraryFrames', 'SOURCE_CONTEXT_SPAN_LIBRARY_FRAMES', 0],
```
Comment (watson) [inline test/config.js]: s/CONTEXT/LINES/

(also on the 3 lines above)

Comment (Qard) [reply test/config.js]: fixed

File: lib/config.js (hunk 2 above)
Comment (Qard) [reply lib/config.js]: fixed

File: lib/config.js (hunk 1 above)
Comment (Qard) [reply lib/config.js]: fixed

**Comment by Qard** [comment]
Good catch. Fixed.

**Comment by Qard** [comment]
Weird. Not sure how I missed `ELASTIC_APM_VERIFY_SERVER_CERT`.

Anyway, it should *actually* be fixed this time. I did a grep for "context" and "validate", neither of which appear anywhere.

**Pull request merged**